backend/main.py

```
@@ -347,11 +350,29 @@ async def get_resource_hub_content():
     try: return await content_agent.fetch_resources()
     except Exception as e: logger.error(f"Error in /resources: {e}", exc_info=True); raise HTTPException(status_code=500, detail=str(e))
 
-@app.get("/api/v1/map/hotspots", response_model=List[schemas.MapHotspot], tags=["Map Data"])
+map_router = APIRouter(prefix="/api/v1/map", tags=["Map Data"])
+
+@map_router.get("/hotspots", response_model=List[schemas.MapHotspot])
```

Flake8 (7.2.0)

[error] expected 2 blank lines, found 1

(E302)